mesa_geo/geospace.py

```diff
@@ -96,7 +94,7 @@ def get_intersecting_agents(self, agent, other_agents=None):
         return intersecting_agents
 
     def get_neighbors_within_distance(
-        self, agent, distance, center=False, relation="intersects"
+            self, agent, distance, center=False, relation="intersects"
```

Comment:
This is an unnecessary indentation.

Reply:
Thank you for spotting this! It's probably due to the auto formating tool from my IDE.